File: KufarCodeApp/Domain/Models/Ad.swift
import Foundation

struct Ad: Identifiable {
    let id: String
    let title: String
    let price: Double?
    let cityCode: String
    let seasonCode: String?
    let categoryId: String?
    let location: GeoPoint?
}

struct GeoPoint {
    let latitude: Double
    let longitude: Double
}

File: android/app/src/main/java/com/kufarcode/app/domain/model/Ad.kt
package com.kufarcode.app.domain.model

data class Ad(
    val id: String,
    val title: String,
    val description: String?,
    val price: Double?,
    val currency: String?,
    val cityCode: String?,
    val imageUrl: String?,
    val latitude: Double?,
    val longitude: Double?,
    val seasonCode: String?
)
